### DISCUSSION TITLE
Bug: No way to install original Fig application

### DISCUSSION THREAD

**Discussion by DemianParkhomenko** [aws/codewhisperer-command-line-discussions#5] [Support ticket]
### Checks

- [X] I ran `cw doctor` in the affected command line session and it didn't resolve my issue
- [X] I ran `cw restart` and it didn't resolve my issue
- [X] I searched [the discussions in this repo](https://github.com/aws/codewhisperer-command-line-discussions/discussions?q=) and there are no duplicates of my issue

### Expected behaviour

I expected to be able to download original Fig application.

### Actual behaviour

It downloads CodeWhisper. 

### Steps to reproduce

1. Go to https://fig.io/download

2. Click on 
![image](https://github.com/aws/codewhisperer-command-line-discussions/assets/95881717/00db4974-9499-45b2-84bf-d7ec476d9bbb)

3. It downloads from https://desktop-release.codewhisperer.us-east-1.amazonaws.com/latest/CodeWhisperer.dmg

**Comment by grant0417** [comment]
Hey thanks for reporting, this should be fixed. Also just so you know signups are currently disabled for the original Fig app if you didn't sign up in the past.
